server/routerlicious/packages/services-ordering-zookeeper/src/zookeeperClient.ts

```diff
@@ -4,28 +4,21 @@
  */
 
 import { IZookeeperClient } from "@fluidframework/server-services-core";
-import * as zookeeper from "node-zookeeper-client";
-
-const RetryAttemps = 3;
```

Comment:
There is no built in retry mechanism in `zookeeper`. It appears to run successfully without these retries.